job-portal/routes/auth.js

```diff
@@ -21,7 +35,11 @@ router.post('/register', function (req, res) {
     var newUser = new User({
       email: req.body.username,
       password: req.body.password
+      
     })
+    if (passValidation(newUser.password) == false){
+      return res.status(409).json({success: false, msg: 'Password requirements not met.'})
```

Comment:
This is a **`400`** error and not a `409`. You can look up the 2xx, 3xx, 4xx, and 5xx error codes online to see what they mean.